dockerfiles/base/scripts/base/docker.sh

```diff
@@ -251,16 +246,11 @@ check_docker_networking() {
   export no_proxy=$NO_PROXY
 }
 
-check_tty() {
+check_interactive() {
+
   # Detect and verify that the CLI container was started with -it option.
-  if [[ ! -t 1 ]]; then
-    info "Welcome to ${CHE_FORMAL_PRODUCT_NAME}!"
-    info ""
-    info "We did not detect a valid TTY."
-    info ""
-    info "TTY Syntax:"
-    info "    Add '-it' to your 'docker run' command."
-    return 2
+  if [ ! -t 1 ]; then
+    warning "Did not detect TTY - interactive mode disabled"
```

Comment:
![image](https://cloud.githubusercontent.com/assets/436777/21092880/35f63600-c04f-11e6-9b97-6f1428bdfb8e.png)

my issue is that user may not understand or see the warning about interactive mode disabled because when he will start the cli he may think cli is behaving badly.

Maybe it's not about checking interactive mode but about detecting result of the read option when asking for the license